### ISSUE TITLE
Better Pushback / Xplane 11 not working right.

### ISSUE THREAD

**Issue by GreggR1** [skiselkov/BetterPushbackC#251]
Better Pushback was woking fine with release of Xplane 11 before 11.30 and Zibo mod 3.31, however when I then opted in for xplane beta 11.30 and went upto next zibo mod pdate 3.32 the aircraft icon and mouse don't line up in pre-plan or request pushback. 
![image](https://user-images.githubusercontent.com/46577371/50995126-0c387080-1516-11e9-84ad-7b57090f1a15.png)

**Comment by blee254** [comment]
I’m having the same problem.  Makes BP useless. Of course the question is whether it’s caused by a ‘permanent’ change with X-Plane or is a transient issue that will be resolved with the final 11.30 release. We shall see.

**Comment by GreggR1** [comment]
well one thing i haven't tried on my setup is setting Xplane 11 to windowed mode currently it's full screen it might be a miracle and that might fix it

**Comment by blee254** [comment]
Good idea. I’ll try that this weekend.

**Comment by GreggR1** [comment]
ive fixed it

**Comment by blee254** [comment]
How? Was it the full screen/windowed mode?

**Comment by blee254** [comment]
I’ve  only run X-Plane 11 in full screen mode (no switching). But at least have something to experiment with. Thanks for the info.

**Comment by GreggR1** [comment]
Yeah so set your sim to Window and see where it sends the window and go
into a flight try BP then if that doesn't not sure what's wrong with yours

I don't think you'll have had to go into windowed to cause the issue it
might when whenever you or if you've move screens around if you have multi
monitor set up in display settings that for me seemed to be what the issue
was

**Comment by blee254** [comment]
So setting X-Plane to windowed mode allowed the pushback planning mode to
work correctly (meaning BetterPushback new where the plane really was and
allowed me to plan the pushback route with no problems).  So then I just
returned to fullscreen mode and the pushback procedure itself worked just
fine.  So while it's a bit of a hassle, it's a problem that at least has a
workaround.

**Comment by GreggR1** [comment]
See now for me, windowed mode works fine but full screen it then returns to
not working right

**Comment by tropicflite** [comment]
As of today May 13, 2020 the issue persists. I'm able to use the plugin to set up the pushback in windowed mode only. I can call for the push in full screen though.

**Comment by Roland95** [comment]
Hello, 
In your last version of Betterpusback the coordinate inversion described below(*)  should be solved, but for me I still have the problem with ZIBO738 for example on Paris CDG (but it's the same on other airports) .  See the attached file below .
Soft Version - Xplane 11.55 / Better Pusback 0.48
Thanks for your comments.
(*)Implemented workaround for when X-Plane returns inverted-Y coordinate at certain sceneries using custom terminal buildings with inverted normals on the roof.


[Pushback_Plan_error.docx](https://github.com/skiselkov/BetterPushbackC/files/7574335/Pushback_Plan_error.docx)
